components/nimbus/src/evaluator.rs

```diff
@@ -31,70 +29,6 @@ impl Bucket {
     }
 }
 
-#[cfg(feature = "stateful")]
-#[derive(Serialize, Deserialize, Debug, Clone, Default)]
-pub struct TargetingAttributes {
-    #[serde(flatten)]
-    pub app_context: AppContext,
-    pub language: Option<String>,
-    pub region: Option<String>,
-    pub is_already_enrolled: bool,
-    pub days_since_install: Option<i32>,
-    pub days_since_update: Option<i32>,
-    pub active_experiments: HashSet<String>,
-    pub enrollments: HashSet<String>,
-}
-
-#[cfg(not(feature = "stateful"))]
```

Comment:
I was very confused why these were being included in `stateless/evaluator.rs` until I read it the 10th time and realized that there is a `not()` here 😂 All the more reason for this separation to happen!

Reply:
Yeah that's exactly why I wanted to do this haha